### ISSUE TITLE
Push Command Fails

### ISSUE THREAD

**Issue by Tornquist** [webpro/release-it#384]
## Description

I have a repo set up with a single remote. `origin` is set using the normal `ssh` git configuration `git@github.com:Tornquist/Time-Core.git`. 

I am able to successfully run `git push` and `git push origin`. Locally I've had no issues with my git config or using git.

I just set it up with `release-it` with a config of:

```
{
  "src": {
    "tagName": "v%s",
    "commitMessage": "Release v%s",
    "tagAnnotation": "Release v%s"
  },
  "github": {
    "release": true
  },
  "npm": {
    "publish": false
  }
}
```

When I run the command, everything is committed and tagged locally, as expected.

During the push the following command is executed:

```
git push --follow-tags git@github.com:Tornquist/Time-Core.git
```

Which does not have a branch specified. It is rejected by git with the following error: `you are pushing to remote which is not the upstream of your current branch`

This is a remote, and it is upstream. However the generated command is not. 

## Question

Is there an expected git configuration to make all this work? Do you see something that I'm doing wrong?

It seems like the check for `/` when generating the push command may be invalid. If it had generated `git push --follow-tags origin`, I believe everything would have worked correctly.

**Comment by Tornquist** [comment]
Based on the tests outlined [here](https://github.com/webpro/release-it/blob/master/test/git.js#L83), the current behavior is exactly as expected.

**Comment by Tornquist** [comment]
From what I can see:

```
valid: git push origin
valid: git push http://....
invalid: git push git@....
```

based on https://git-scm.com/docs/git-push#_git_urls_a_id_urls_a

**Comment by webpro** [comment]
Thanks for the detailed feedback, @Tornquist! (#306 and #379 are related).

I've just released [7.6.3-beta.0](https://github.com/webpro/release-it/releases/tag/7.6.3-beta.0) that should fix the issue. Could you please verify this works for you as well? Thanks in advance!

**Comment by Tornquist** [comment]
That worked for me. Everything looks good from my end now. 👍
